docs-chef-io/content/workstation/troubleshooting.md

```diff
@@ -20,6 +20,10 @@ Chef Workstation logs are stored in ` ~/.chef-workstation/logs`.
 
 Follow the steps provided under [Uninstalling]({{< ref "install_workstation.md#uninstalling" >}}).
 
+## Trusted Certs
+
+We advise restarting the Chef Workstation Powershell after adding certificates to the `trusted_certs` directory on Windows machines. Sometimes certificate-related commands such as `knife ssl check` don't return the expected results after adding a certificate with `knife ssh fetch`. If this happens, exit the Chef Workstation Powershell, select the icon, and retry the command.
```

Comment:
by restart do we just mean open a new PS prompt?

Reply:
@tas50 kgarmoe made some edits that I just added that likely address this question.